src/abi/aavev3stata/Factory.json

```diff
@@ -0,0 +1,128 @@
+[
```

Comment:
Unused file

Reply:
It's used [here](https://github.com/paraswap/paraswap-dex-lib/pull/700/files#diff-5d7c647f1faef6ba75b678ad3eb6d13839904f90b91e91e53f0b94a07567d8abR73-R76)